Makefile

```diff
@@ -204,3 +205,6 @@ cluster-logging-operator-install:
 # uninstalls the cluster-logging operator
 cluster-logging-operator-uninstall:
 	olm_deploy/scripts/operator-uninstall.sh
+
+check-go-version:
```

Comment:
I suggest to add this command to the `.PHONY` targets as it is not relevant for file/dirname expansion by make.

Reply:
done